### DISCUSSION TITLE
Shapely Installation on Windows

### DISCUSSION THREAD

**Discussion by Czarified** [robbievanleeuwen/section-properties#74] [General]
In PR #69 , @connorferster  mentioned shapely being difficult to install on Windows. I've not experienced this so far. In the [documentation ](https://shapely.readthedocs.io/en/stable/project.html#requirements)for Shapely, it mentions that a built distribution including GEOS is now available with a modern version of pip.

This is the approach I'm using, and I can run basic shapely code like the following in a notebook, with graphical geometry output.

```python
import sys
print(sys.executable)
print(sys.version)
from shapely.geometry import Point
patch = Point(0.0, 0.0).buffer(10.0)
print(patch.area)
patch
```

> c:\users\bcrews\.venv\base\scripts\python.exe
3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)]
313.6548490545939
<shapely.geometry.point.Point object at 0x...> (this is shown graphically in the notebook)

Do other users on Windows have problems as Connor described?

**Comment by Agent6-6-6** [comment]
I vaguely remember having to manually find and install geos. But was several years ago when I first installed shapely, so memory is a bit gone on exactly what I had to do.

But I seem to remember once I figured out that that I had no issues.

Same with meshpy, rather than going with the link in the docs, I installed it from a repository for meshpy. The wheels linked to were not available for higher versions of python last time I looked at the link in the readme doc. So you had to go down this route. Wasn't really a problem to sort.

**Comment by normanrichardson** [comment]
I created a new virtual environment and installed Shapely (just using `pip install Shapely`) and had no issues running:
```python
import sys
print(sys.executable)
print(sys.version)
from shapely.geometry import Point
patch = Point(0.0, 0.0).buffer(10.0)
print(patch.area)
patch
```
Let me know if any other details will help.
